src/app/components/ParaTimePicker/ParaTimeMenu.tsx

```diff
@@ -0,0 +1,100 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import Typography from '@mui/material/Typography'
+import KeyboardArrowRightIcon from '@mui/icons-material/KeyboardArrowRight'
+import MenuList from '@mui/material/MenuList'
+import MenuItem from '@mui/material/MenuItem'
+import ListItemText from '@mui/material/ListItemText'
+import { COLORS } from '../../../styles/theme/colors'
+import { Layer } from '../../../oasis-indexer/api'
+import { getLayerLabels } from '../../utils/layout'
+
+type ParaTimeMenuItemProps = ParaTimeMenuProps & {
+  divider: boolean
+  enabled: boolean
+  layer: Layer
+}
+
+export const ParaTimeMenuItem: FC<ParaTimeMenuItemProps> = ({
+  activeLayer,
+  divider,
+  enabled,
+  hoveredLayer,
+  layer,
+  selectedLayer,
+  setHoveredLayer,
+  setSelectedLayer,
+}) => {
+  const { t } = useTranslation()
+  const labels = getLayerLabels(t)
+
+  return (
+    <>
+      <MenuItem
+        divider={divider}
+        sx={{
+          cursor: enabled ? 'pointer' : 'default',
+        }}
+        onMouseEnter={() => {
+          if (layer !== selectedLayer) {
+            setSelectedLayer(undefined)
+          }
+          setHoveredLayer(layer)
+        }}
+        onClick={enabled ? () => setSelectedLayer(layer) : undefined}
+      >
+        <ListItemText>
+          {labels[layer]}
+          <Typography
+            component="span"
+            sx={{ fontSize: '10px', fontStyle: 'italic', color: COLORS.grayMedium, ml: 2 }}
+          >
+            {activeLayer === layer && t('paraTimePicker.selected')}
```

Comment:
Bug:
- open picker on testnet https://d35c4db7.oasis-explorer.pages.dev/testnet/emerald
- it shows Emerald Mainnet as currently selected
![image](https://github.com/oasisprotocol/explorer/assets/3758846/db0d8f81-c633-4fd6-a47a-49a355fdc9d0)